hubbard/hamiltonian.py

```diff
@@ -55,20 +54,25 @@ def __init__(self, TBHam, n=0, U=0.0, q=(0., 0.), nkpt=[1, 1, 1], kT=1e-5):
         self.H = TBHam.copy()
         self.H.finalize()
         self.geometry = TBHam.geometry
+        # So far we only consider either unpolarized or spin-polarized Hamiltonians
+        self.size = self.H.spin.spinor
```

Comment:
Without further labelling I find `size` not so descriptive. Maybe `self.spinor` or `self.spin_size` would be clearer?

Reply:
Yeah, maybe is better to call it `self.spin_size`. To me `spinor` sounds like it could be confusing. I will change then the variable name.